src/test/java/gov/healthit/chpl/aqa/asserts/ViewDeveloperPageAsserts.java

```diff
@@ -0,0 +1,102 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.ViewDeveloperPage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+
+/**
+ * Class ViewDeveloperPageAsserts definition.
+ */
+public class ViewDeveloperPageAsserts extends Base {
+
+    /**
+     * Assert that developer edit link exists on view developer page.
+     * @param developerId is the developer id passed in page objects
+     */
+    @Then("^I see developer edit link for developer \"([^\"]*)\"$")
+    public void iSeeDeveloperEditLink(final String developerId) {
+        try {
+            assertTrue(ViewDeveloperPage.editDeveloperLink(getDriver(), developerId).isDisplayed());
+        } catch (NoSuchElementException e) {
+            fail("Unable to find developer edit link");
+        }
+    }
+
+    /**
+     * Assert that developer split link exists on view developer page.
+     * @param developerId is the developer id passed in page objects
+     */
+    @Then("^I see developer split link for developer \"([^\"]*)\"$")
+    public void iSeeDeveloperSplitLink(final String developerId) {
+        try {
+            assertTrue(ViewDeveloperPage.splitDeveloperLink(getDriver(), developerId).isDisplayed());
+        } catch (NoSuchElementException e) {
+            fail("Unable to find developer split link");
+        }
+    }
+
+    /**
+     * Assert that developer merge link exists on view developer page.
+     * @param developerId is the developer id passed in page objects
+     */
+    @Then("^I see developer merge link for developer \"([^\"]*)\"$")
+    public void iSeeDeveloperMergeLink(final String developerId) {
+        try {
+            assertTrue(ViewDeveloperPage.mergeDeveloperLink(getDriver(), developerId).isDisplayed());
+        } catch (NoSuchElementException e) {
+            fail("Unable to find developer merge link");
+        }
+    }
+
+    /**
+     * Assert that developer name is correct.
+     */
+    @Then("^I see the edited developer information is recorded and updated on View Developer page$")
+    public void editedDeveloperInformationRecordedAndUpdated() {
+        String actualDeveloperVersion = ViewDeveloperPage.viewDeveloperName(getDriver()).getText();
```

Comment:
What is `Version` doing here? Does this assert have anything to do with Versions?